build.zig

```diff
@@ -25,7 +25,8 @@ const aarch64_cortexa53 = CrossTarget{
 };
 
 pub fn build(b: *Builder) !void {
-    const target = b.standardTargetOptions(.{ .whitelist = &[_]CrossTarget{ x86_i686, aarch64_cortexa53 }, .default_target = x86_i686 });
+    // target changed temporarily because -Dtartet=cortex_a53-freestanding produces unknown architecture
+    const target = b.standardTargetOptions(.{ .whitelist = &[_]CrossTarget{ x86_i686, aarch64_cortexa53 }, .default_target = aarch64_cortexa53 });
```

Comment:
To build for aarch64 you should pass "-Dtarget=aarch64-freestanding", rather than specify the CPU name.

Reply:
K